e2e/docker-compose.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-COMPOSE_DOCKER_CLI_BUILD=1 docker compose --file ${projectRoot}/e2e/docker-compose-workdir.yaml --file ${projectRoot}/docs/docs/guides/deploy/docker-compose.yaml --file ${projectRoot}/e2e/docker-compose-overwrite.yaml "$@"
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
I don't like that we need scripts with such logic to it

Reply:
we could move it to the console/cypress folder, the compose yamls are the only files left in the e2e folder anyway.
When we move the command to the npm script, the path is already deterministic and we don't need this line for the SCRIPTPATH anymore.

Reply:
is this file still needed?

Reply:
nope, I deleted it